.github/workflows/build-test-image.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Test Image
+
+on:
+  schedule:
+    - cron: "0 03 * * *" # run every day at 03:00 UTC
```

Comment:
Do you think we need to have a scheduled rebuild of the image if it also will be built in a PR?

Reply:
no I think if we're building on hash updates then it should be okay 👍🏼